docassemble/juvenilesealing/data/questions/entrypoint-petitioner.yml

```diff
@@ -146,7 +146,7 @@ subquestion: |
 
   You need to know about your closed cases and open cases.
   That includes cases in other states and federal cases.
-  Having {a copy of your criminal records}
+  Having {a copy of your criminal records <i class="fa fa-question-circle" aria-hidden="true"></i>}
```

Comment:
Can you put this icon in a variable like we do with `next_icon`? That will make it easier to understand, re-use and change in the future

Reply:
Great idea! We can work on this on Tuesday.

Reply:
It was not possible to do -- we left it as it was (question circle worked)